src/test/java/cs361/battleships/models/BoardTest.java

```diff
@@ -145,4 +145,43 @@ public void testSonarPulseRightAmountOfSquaresSides() {
         board.sonarPulse(5, 'J');
         assertEquals(board.getAttacks().size(), 36);
     }
+
+    @Test
+    public void testMoveCorrectDirection() {
+        board.placeShip(new Ship("MINESWEEPER"), 6, 'D', true);
```

Comment:
Good thorough tests for each direction, increases confidence of code functionality